source/features/release-download-count.tsx

```diff
@@ -78,12 +78,23 @@ async function init(): Promise<void | false> {
 				}
 
 				// Place next to asset size
-				assetName
+				const assetSize = assetName
 					.closest('.Box-row')!
-					.querySelector('.float-right')!
+					.querySelector(':scope > .flex-justify-end > :first-child')!;
+
+				const classes = [
+					'rgh-release-download-count',
+					...assetSize.classList,
+				].map(name => name.replace('text-sm-left', 'text-sm-right'));
+
+				if (index === 0) {
+					classes.push('text-bold');
+				}
+
+				assetSize
 					.after(
```

Comment:
```suggestion
				assetSize.after(
```

Reply:
Prettier made me forget I need to handle this stuff manually here 🥲